app/assets/javascripts/angular/application.js.erb

```diff
@@ -1,10 +1,18 @@
-var app = angular.module('Roomi', ['ngRoute']);
+var app = angular.module('Roomi', ['ngRoute', 'ng-token-auth']);
 
-app.config(['$routeProvider', function($routeProvider) {
+app.config(['$routeProvider','$locationProvider','$authProvider', function($routeProvider,$locationProvider, $authProvider) {
   // Configure Angular Routes
   $routeProvider.
     when('/', {
       templateUrl: "<%= asset_path('static_pages/index.html') %>",
       controller: 'StaticPagesCtrl'
+    }).
+    when('/signup', {
+      templateUrl: "<%= asset_path('auth/signup.html') %>",
+      controller: 'SignUpCtrl'
     })
+
+  $authProvider.configure({
+  apiUrl: 'http://localhost:3000'
```

Comment:
Don't forget to put the rails environment variable for your app to work on Heroku